gradle-metric-schema/src/test/groovy/com/palantir/metric/schema/gradle/MetricSchemaPluginIntegrationSpec.groovy

```diff
@@ -79,6 +79,25 @@ class MetricSchemaPluginIntegrationSpec extends IntegrationSpec {
         """.stripIndent()
     }
 
+    def 'handles shorthand schema'() {
+        when:
+        file('src/main/metrics/metrics.yml') << """
+        namespaces:
+          server:
+            docs: General web server metrics.
+            metrics:
+              response.size:
+                type: histogram
+                docs: A histogram of the number of bytes written into the response.
+                tags:
+                  - tagName: serviceName
```

Comment:
nit: Thoughts on `name` since `tag` context is provided by the parent set?

Reply:
done